typescript/testsuite/testsuite_impl/advanced_network_test/advanced_network_test_.ts

```diff
@@ -0,0 +1,75 @@
+import { NetworkContext, Network } from "kurtosis-core-api-lib";
```

Comment:
Note about filenames: the trailing `_` was necessary in Go because files ending in `_test` are automatically declared test files in Go. We don't need this in Typescript, so you can remove the `_` before `.ts` in all these file names